modal/config.py

```diff
@@ -268,7 +268,7 @@ def __repr__(self):
         return repr(self.to_dict())
 
     def to_dict(self):
-        return {key: self.get(key) for key in _SETTINGS.keys()}
+        return {key: self.get(key) for key in sorted(_SETTINGS)}
```

Comment:
Not directly related but when confirming that we can still set a valid environment I noticed that the `modal config show` output was hard to read because it was unordered.